integration-tests/utils/integration-helpers.js

```diff
@@ -80,6 +80,17 @@ const addUserToBilling = _.flow(withSignedInPage, withUserToken)(async ({ page,
   }, email, billingProject)
 
   console.info(`added user to: ${billingProject}`)
+
+  const userList = await page.evaluate((billingProject) => {
+    return window.Ajax().Billing.project(billingProject).listUsers()
+  }, billingProject)
+
+  const getVal = email => {
+    const userKey = _.find({email}, userList)
+    return userKey.role
+  }
+
+  console.info(`test user was added to the billing project with the role: ${getVal(email)}`)
```

Comment:
Why not just simplify this?
```suggestion
  const billingUser = _.find({ email }, userList)

  console.info(`test user was added to the billing project with the role: ${!!billingUser && billingUser.role}`)
```

Reply:
made this change. very elegant suggestion!